### DISCUSSION TITLE
accessing a restricted user's profile should display that the user is restricted

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#16864] [Ideas] [answered]
currently, when accessing a restricted user's profile, the profile will simply not load(but the previous profile loaded will remain visible) 

what i think it should do is replicate the website's design so that the user knows that said person has been restricted

**Comment by Theighlin** [comment, marked as answer]
#8722

**Reply by iSimpFor** [reply]
i didn't quite catch that when i was checking for where the issue was already mentioned, good catch

**Reply by Theighlin** [reply]
It was less a good catch and more like I made the same mistake
